app/models/katello/content_view.rb

```diff
@@ -76,6 +74,7 @@ def <<(*_args)
 
     scoped_search :on => :name, :complete_value => true
     scoped_search :on => :organization_id, :complete_value => true
+    scoped_search :on => :composite, :complete_value => true
```

Comment:
I might could see also adding searching by repositories, puppet module names, and environments (via their version)